packages/components/src/tip/stories/index.js

```diff
@@ -0,0 +1,13 @@
+
+/**
+ * Internal dependencies
+ */
+import Tip from '../';
+
+export default { title: 'Components|Tip', component: Tip };
+
+export const _default = () => (
+	<Tip>
+		<p>An example tip.</p>
```

Comment:
@mkaz Thanks for your continued Storybook support! <3

As an enhancement, I think it would be helpful to add a `text()` knob for this content. What do you think? :)

Reply:
@ItsJonQ Good suggestion, added. :+1: